autoload/LanguageClient.vim

```diff
@@ -1502,4 +1512,78 @@ function! LanguageClient#debugInfo(...) abort
     return LanguageClient#Call('languageClient/debugInfo', l:params, l:Callback)
 endfunction
 
+function! s:ClosePopups() abort
+  if s:ShouldUseFloatWindow()
+    call s:CloseFloatingHover()
+  elseif s:POPUP_WINDOW_AVAILABLE && s:GetVar('LanguageClient_usePopupHover', v:true)
+    call popup_clear()
+  else
+    :pclose
+  endif
+endfunction
+
+function! LanguageClient#showCompletionItemDocumentation(...) abort
+  if !has_key(v:completed_item, 'user_data')
+    call s:ClosePopups()
+    return
+  endif
+
+  if v:completed_item['user_data'] ==# ''
+    call s:ClosePopups()
+    return
+  endif
+
+  let l:user_data = json_decode(v:completed_item['user_data'])
+  let l:completed_item = {}
+
+  if has_key(l:user_data, 'ncm2_lspitem')
```

Comment:
I'm not sure why this is needed really, all I know is that the `lspitem` key is `ncm2_lspitem` when running `ncm2`, but I'm not sure if other plugins (deoplete, etc) do this as well or not.